packages/relay/src/StoredTransaction.ts

```diff
@@ -25,7 +25,8 @@ export interface StoredTransactionMetadata {
 export interface StoredTransactionSerialized {
   readonly to: Address
   readonly gas: number
-  readonly gasPrice: number
+  maxFeePerGas: number
+  maxPriorityFeePerGas: number
```

Comment:
you can actually easily keep these as `readonly` with 2 variables (for consistency and extra protection)
you can also avoid `details as StoredTransactionSerialized` this way, I think.